crates/bevy_gltf/src/loader.rs

```diff
@@ -252,23 +280,10 @@ async fn load_gltf<'a, 'b>(
 fn load_material(material: &Material, load_context: &mut LoadContext) -> Handle<StandardMaterial> {
     let material_label = material_label(&material);
     let pbr = material.pbr_metallic_roughness();
-    let mut dependencies = Vec::new();
     let texture_handle = if let Some(info) = pbr.base_color_texture() {
-        match info.texture().source().source() {
-            gltf::image::Source::View { .. } => {
-                let label = texture_label(&info.texture());
-                let path = AssetPath::new_ref(load_context.path(), Some(&label));
-                Some(load_context.get_handle(path))
-            }
-            gltf::image::Source::Uri { uri, .. } => {
-                let parent = load_context.path().parent().unwrap();
-                let image_path = parent.join(uri);
-                let asset_path = AssetPath::new(image_path, None);
```

Comment:
With this PR no longer using the asset system to load external texture files (and not tracking the image path / not using it for the handle), and not adding them as asset dependencies, I suspect that might mean that the asset system is no longer going to detect if any of the referenced texture files is modified, which means that they will not be hot-reloaded.

(Instead, modifying the GLTF itself would now hot-reload everything, with all the external textures, as they are treated the same as embedded GLB textures.)

Further, we lose the performance/parallelism benefits that we had before, when textures were queued up to be loaded asynchronously by the asset system.

Reply:
Without a large rework, the asset server is not able to load this kind of assets with external configuration. I think losing  the hot-reload of parts of a gltf in exchange for it being correctly loaded is a gain.

Longer term, the Distill integration should provide this feature.

For the async part, I have something working locally (that still need work), but would prefer this and #1632 merged first to avoid having too many pr touching the same parts at the same time.

Reply:
Yeah, the current asset system is kinda limited and quirky, really looking forward to Distill. For now we have to make do with what we have, and I agree that loading and rendering the assets correctly should take priority over the features of the asset system working smoothly.

I am excited about your improvement/fix for async loading, sounds like it should address the performance issues when loading big GLBs with many large textures.

Reply:
Yeah I think this is the right short term move. I don't think async will solve the performance issues with image loading. We aren't queuing up multiple async tasks at the same time, so while it might run the async task on a separate thread, we won't start the next one until the previous one is finished.

Reply:
Ahh nobody was claiming the current async stuff was a perf improvement. My bad!